frappe/config/setup.py

```diff
@@ -136,6 +136,11 @@ def get_data():
 				},
 				{
 					"type": "doctype",
+					"name": "Email Domain",
```

Comment:
seems wierd to have 2 services
the options currently are use service or configure a domain that will autoselect
i tried service in domain but domain kinda requires an incoming and outgoing account which some services dont have